src/lookup-table/controllers/ExportCsv.controller.ts

```diff
@@ -5,7 +5,7 @@ import { errorMessage } from '../../components/utilityFunctions'
 const ExportCsv = async (request: any, reply: any) => {
   const tableId = Number(request.params.lookupTableId)
 
-  const lookupTableService = await LookupTableService({ tableId })
+  const lookupTableService = await LookupTableService({ tableId, dataViewCode: 'ignore' })
```

Comment:
Ugly, but better than making `dataViewCode` optional. (I would refactor this substantially if I had the chance)